<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="TabWdgtSettings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>451</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="TabDataBase">
    <attribute name="title">
     <string>База данных</string>
    </attribute>
    <widget class="QGroupBox" name="GrpBoxDataBaseDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>База данных</string>
     </property>
     <widget class="QLabel" name="LblDataBaseDir">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Расположение:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EditDataBaseDir">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="BtnFindDataBaseDir">
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnOpenDataBase">
         <property name="text">
          <string>Открыть базу данных</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="TabCategory">
    <attribute name="title">
     <string>Категории</string>
    </attribute>
    <widget class="QGroupBox" name="GrpBoxStaffer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Сотрудники</string>
     </property>
     <widget class="QComboBox" name="CmbBoxStaffer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAddStaffer">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRemoveStaffer">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GrpBoxCategory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Категории</string>
     </property>
     <widget class="QComboBox" name="CmbBoxCategory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAddCategory">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRemoveCategory">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GrpBoxTax">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>611</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Виды налогов</string>
     </property>
     <widget class="QComboBox" name="CmbBoxTax">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAddTax">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRemoveTax">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="BtnAddDefCategory">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>390</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnRemoveDefCategory">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>390</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LblDefCategoryTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Заданные категории:</string>
     </property>
    </widget>
    <widget class="QTableView" name="TableViewDefCategory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>611</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TabCoeff">
    <attribute name="title">
     <string>Коэффициенты</string>
    </attribute>
    <widget class="QLabel" name="LblBasicWage">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Основная ставка сотрудника:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditBasicWage">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditBasicWageKoefSalary">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LblBasicWageKoefSalary">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Коэффициент ставки сотрудника:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
